pages/common/gh-browse.md

```diff
@@ -0,0 +1,36 @@
+# gh browse
+
+> Open a GitHub repository in the browser or print the URL.
+> More information: <https://cli.github.com/manual/gh_browse>.
+
+- Open the home page of the current repository in the default web browser:
```

Comment:
I think we should use `homepage` (https://trends.google.com/trends/explore?geo=AT&q=home%20page,homepage)
```suggestion
- Open the homepage of the current repository in the default web browser:
```